yarn-project/blob-sink/src/metrics.ts

```diff
@@ -16,7 +16,7 @@ export class BlobSinkMetrics {
     });
 
     this.blobSize = telemetry.getMeter(name).createHistogram(Metrics.BLOB_SINK_BLOB_SIZE, {
-      description: 'The size of blobs in the blob store',
+      description: 'The non zero size of blobs in the blob store',
```

Comment:
Think non-padded would convey it better. Or the size of non-zero elements or something. But guess you can understand so probably fine.